### PR TITLE
Diagnostic responses

### PR DESCRIPTION
**Pull request by cuprous**

### CODE REVIEW

File: lib/diagnostic.rb [hunk 1]
```diff
@@ -115,10 +117,10 @@
 # Use shorthand to make this one line long.
 # Write your response as a Ruby string.
 
-Diagnostic.person_hash = 'your answer here'
+Diagnostic.person_hash = "person = {first_name: \"Zachary\", age: 29}"
```
Comment (J-Weeks) [inline lib/diagnostic.rb]: person = {first_name: "Jason", age: 27}'
Why the added breaks?

File: lib/diagnostic.rb [hunk 2]
```diff
@@ -115,10 +117,10 @@
...
 
 # Set a default return value to your `person` hash that concatenates the value
 # of person[:first_name] and a message about them.
 # Write your response as a Ruby string.
 
-Diagnostic.hash_default = 'your answer here'
+Diagnostic.hash_default = "person.default = \"\#{person[:first_name]}"
```
Comment (J-Weeks) [inline lib/diagnostic.rb]: same here

File: lib/diagnostic.rb (hunk 1 above)
Comment (cuprous) [inline lib/diagnostic.rb]: Are you referring to the escaped quotes? If so, rspec broke without them.

**Comment by J-Weeks** [comment]
17/17. Good job.

**Pull request closed without merging**